src/controllers/admin.controller.js

```diff
@@ -1,22 +1,24 @@
-adminCtrl.renderIndex = async () => {
+const adminCtrl = {};
+
+adminCtrl.renderIndex = async(req, res) =>{
 };
 
-adminCtrl.renderUserList = async () => {
+adminCtrl.renderUserList = async(req, res) =>{
 }; 
 
-adminCtrl.renderAddUserForm = async () => {
+adminCtrl.renderAddUserForm =async(req, res) =>{
```

Comment:
Take care with this kind of changes (blank spaces), please be sure you have these plugins:
Integrated Terminal 
Prettier 
Material Icon Theme